pkgs/development/compilers/kotlin/default.nix

```diff
@@ -1,43 +1,100 @@
 { lib, stdenv, fetchurl, makeWrapper, jre, unzip }:
 
-stdenv.mkDerivation rec {
-  pname = "kotlin";
+let
   version = "1.5.30";
+  homepage = "https://kotlinlang.org/";
+  license = lib.licenses.asl20;
+  description = "A modern programming language that makes developers happier";
+  longDescription = ''
+    Kotlin is a modern but already mature programming language aimed to make
+    developers happier. It’s concise, safe, interoperable with Java and other
+    languages, and provides many ways to reuse code between multiple platforms
+    for productive programming.
+  '';
+in {
```

Comment:
Why are those variables in the let in block?

Reply:
As duplication and being readable upon opening the file.